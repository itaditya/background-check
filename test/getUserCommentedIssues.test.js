const getUserCommentedIssues = require('../lib/github-api/getUserCommentedIssues')

test('getUserCommentedIssues is working', async () => {
  const github = {
    search: { issues: jest.fn() }
  }

  await getUserCommentedIssues(github, {
    username: 'itaditya'
  })

  expect(github.search.issues.mock.calls).toMatchSnapshot()
})
