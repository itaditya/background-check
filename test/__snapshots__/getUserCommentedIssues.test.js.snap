// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getUserCommentedIssues is working 1`] = `
Array [
  Array [
    Object {
      "q": "commenter:itaditya",
    },
  ],
]
`;
